README.md

```diff
@@ -93,6 +93,6 @@ having any problems, join our office hours, or open an issue in our GH repo.
 ## Where can I go for help?
 
 - Slack,
-  [#ibmproducts-pal-dev](https://ibm.enterprise.slack.com/archives/C013ZTX0N6B)
+  [#ibmproducts-pal-dev](https://ibm.enterprise.slack.com/archives/CQGR0HC05)
```

Comment:
Good catch! Can you also update the name of the channel? (ie `carbon-for-ibmproducts`)